src/lyra.ts

```diff
@@ -649,11 +648,9 @@ export function search<S extends PropertiesSchema>(
     for (let j = 0; j < tokensLength; j++) {
       const term = tokens[j];
       const documentIDs = getDocumentIDsFromSearch(lyra, { ...params, index, term, exact });
-      
+
       // lyraOccurrencies[term] can be undefined, 0, string, or { [k: string]: number }
-      const termOccurrencies = typeof lyraOccurrencies[term] === "number"
-        ? (lyraOccurrencies[term] ?? 0)
-        : 0;
+      const termOccurrencies = typeof lyraOccurrencies[term] === "number" ? lyraOccurrencies[term] ?? 0 : 0;
```

Comment:
I think this is hard to read maybe its better in extended form

Reply:
That's eslint doing its magic 😂

Reply:
why do you need to chekc if it's a number? I'm thinking this whole row can be replaced by

```js
const termOccurrencies = lyraOccurrencies[term] ?? 0
```